frontends/web/src/locales/en/app.json

```diff
@@ -1013,6 +1013,27 @@
         "title": "Why is there a network fee?"
       }
     },
+    "settings-block-explorer": {
+      "instructions": {
+        "link": {
+          "text": "Guide on how to use a block explorer"
+        },
+        "text": "For a full tutorial, please visit our guide:",
+        "title": "How do I use a block explorer?"
+      },
+      "options": {
+        "text": "The BitBoxApp features a protection mechanism designed to prevent the opening of arbitrary links. This serves as a security measure against malicious links. It cannot be bypassed on an individual basis; rather, its suspension would be required to allow users to configure their own URLs.",
```

Comment:
imo this is super confusing to users, especially the last part. i'd stop at the first part

"The BitBoxApp features a protection mechanism designed to prevent the opening of arbitrary links. This serves as a security measure against malicious links."